app/build.gradle.kts

```diff
@@ -75,7 +75,7 @@ dependencies {
     implementation(libs.kotlinx.serialization.converter)
     implementation(libs.kotlinx.serialization)
     implementation(libs.coil)
-
+    implementation(libs.retrofit.converter.gson)
```

Comment:
use something modern like moshi or kotlinx.serialization

Reply:
Okay. On it